README.md

```diff
@@ -1476,8 +1476,6 @@ Thanks for the course and good luck to students! \
 ### I'm [Vlad](https://github.com/MelmQuty) and I didn't find any typo in ur README(maybe because it's perfect:rocket:) but I'm gonna show your my cat.He is a little impudent.:octocat:
 ![cat](https://raw.githubusercontent.com/MelmQuty/cat/master/cat.jpg)
 
----
-
```

Comment:
Please recover these lines too

Reply:
There is someone's code on lines 1479-1480. Should I move it down and add those lines?